backend/dataall/core/permissions/services/tenant_policy_service.py

```diff
@@ -1,20 +1,13 @@
-import logging
+from dataall.core.permissions.db.tenant.tenant_policy_repositories import TenantPolicy
+from dataall.core.permissions.constants import permissions
+from dataall.core.permissions.db.permission.permission_repositories import Permission
+from dataall.core.permissions.api.enums import PermissionType
+from dataall.base.db import exceptions
+from dataall.core.permissions.db.tenant.tenant_models import TenantPolicy as TenantPolicyModel, TenantPolicyPermission
```

Comment:
Instead renaming the TenantPolicy as TenantPolicyModel, we could rename the TenantPolicy coming fromt enant_policy_repositories as TenantPolicyRepository to keep the same naming convention that we have followed in the rest of the modules